pkg/k8sapi/k8sutils.go

```diff
@@ -68,9 +72,22 @@ func CreateCachedKubeClient(rawK8SClient client.Client, mapper meta.RESTMapper)
 	vpcCniScheme := runtime.NewScheme()
 	clientgoscheme.AddToScheme(vpcCniScheme)
 	eniconfigscheme.AddToScheme(vpcCniScheme)
+	rcscheme.AddToScheme(vpcCniScheme)
 
 	stopChan := ctrl.SetupSignalHandler()
-	cache, err := cache.New(restCfg, cache.Options{Scheme: vpcCniScheme, Mapper: mapper})
+
+	opts := cache.Options{Scheme: vpcCniScheme, Mapper: mapper}
+	if nodeName := os.Getenv("MY_NODE_NAME"); nodeName != "" {
```

Comment:
Is this env var already defined?

Reply:
It is defined in CNI config yaml.
https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/master/aws-k8s-cni.yaml#L204